python/psautohint/autohint.py

```diff
@@ -214,8 +215,11 @@ def openFile(path, outFilePath, options):
 
     if font_format == "UFO":
         font = openUFOFile(path, outFilePath, options)
-    else:
+    elif font_format in ("OTF", "CFF"):
         font = openOpenTypeFile(path, outFilePath, font_format, options)
+    else:
+        log.error("%s format is not supported yet, sorry.", font_format)
+        sys.exit(1)
```

Comment:
I’m not sure if it is a good idea to call `sys.exit()` from the module, this would break other applications (an imaginary future version of `ufo2ft`, for example) importing and calling this module.

Reply:
Yes, SystemExit exceptions only from main() please

Reply:
This is just a temporary solution. Feel free to change it. I can't think of a better way to do it right down. My only request is for it to terminate gracefully from the point of view of a user running psautohint on the command line.

Reply:
We can raise a specific exception, say `ACFontNotSupported` and catch that in main and fail gracefully.

Reply:
sounds good. I won't be able to do it until tomorrow.

Reply:
Or, since this is temporary until Type 1 support is restored, just `NotImplementedError`.